### PR TITLE
Upgrade to node 16, Fabric.js v5.2.1

### PR DESCRIPTION
**Pull request by djmetzle**
Update our Dockerfile and package.json to target Node 16 (on alpine).

This does not work as presented. We will need to stub out the `run-tests` script to prevent CI fails while we iterate.

But this is a step in the right direction! For now, we will tolerate being broken on master, and rally onward with getting the library patched up to work _correctly_ on Node 16.

This PR as written unlocks a few follow improvement PRs.

qa_req 0 (doesnt actually work, just pass CI for now)

CC @andyg0808

### CODE REVIEW

File: package.json
Comment (djmetzle) [inline package.json:23]:
```suggestion
    "fabric": "https://github.com/iFixit/fabric.js.git#try-v521",
```

File: Dockerfile
Comment (klmork) [inline Dockerfile:21]: I'm assuming things listed here that aren't listed above (like perl-Digest-SHA) just aren't needed anymore?

Comment (djmetzle) [reply Dockerfile:21]: Yep! Everything seems to work decently well on `node:16-alpine`. Though these `dnf install`s have been swapped out with analogous `apk add`s up the Dockerfile a bit.

**Comment by klmork** [comment]
Don't know if I'm the best one to approve this, but looks good to me!  Feel free to bump up the cr-req if you want another pair of eyes.

CR 🌸

**Comment by djmetzle** [comment]
All good @klmork thanks! 
But i do need an approval to merge :sweat_smile:

**Review by klmork — APPROVED** [review]

**Comment by klmork** [comment]
Sorry about that!

**Pull request merged**
